packages/gnome-shell/src/ui/modalDialog.d.ts

```diff
@@ -67,22 +74,24 @@ export class ModalDialog extends St.Widget {
      * e.g., if a user clicked "Log Out" then the dialog should go away
      * immediately, but the lightbox should remain until the logout is
      * complete.
-     * @param timestamp 
      */
     protected _fadeOutDialog(timestamp: number): void;
 
+    public vfunc_key_press_event(event: Clutter.Event): boolean;
+    public vfunc_captured_event(event: Clutter.Event): boolean;
+
     public clearButtons(): void;
-    public setButtons(buttons: any[]): void;
-    public addButton(buttonInfo: any): void;
+    public setButtons(buttons: ButtonInfo[]): void;
+    public addButton(buttonInfo: ButtonInfo): St.Button;
     public setInitialKeyFocus(actor: St.Widget): void;
-    public open(timestamp: number, onPrimary: boolean): boolean;
-    public close(timestamp: number): boolean;
+    public open(): boolean;
+    //note: upstream has timestamp as paramater here, but it only will be used to call popModal, which doesn't accept that paramater anymore, so this is just a bug upstream
```

Comment:
Is this issue reported to upstream and being fixed?

If so, why not add a link here?  If not, perhaps report it, to clarify if it's a bug here or with popModal?

Reply:
I intend to do that, but not yet 😓

Reply:
The bug is clearly here, since the last commits seem to get rid of the timestamp parameter everywhere else, they just missed it here

Reply:
made a MR on this issue: [here](https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/3253)